src/frame/js/actions/user.js

```diff
@@ -96,6 +99,14 @@ export function update(props) {
     };
 }
 
+export function _resetState() {
```

Comment:
does it really need to be exported? if it needs to be called from the test, you can use the Rewire API to get it.